src/pages/EditMonitor.vue

```diff
@@ -376,10 +376,10 @@
                                     <label for="method" class="form-label">{{ $t("Method") }}</label>
                                     <select id="method" v-model="monitor.authMethod" class="form-select">
                                         <option :value="null">
-                                            None
+                                            {{ $t("None") }}
                                         </option>
                                         <option value="basic">
-                                            Basic
+                                            {{ $t("Basic") }}
```

Comment:
In my opinion, it should use `{{ $t("HTTP Basic Auth") }}` and the language file remains unchange, because I believe not all languages can be changed like this.

Reply:
> In my opinion, it should use `{{ $t("HTTP Basic Auth") }}` and the language file remains unchange, because I believe not all languages can be changed like this.

I agree that, maybe "Basic" is oversimplified.

Reply:
Sounds good, I'll change back to that.

Reply:
FYI, Microsoft documents are using term "Basic Authentication" ([[1]](https://docs.microsoft.com/en-us/iis/configuration/system.webserver/security/authentication/basicauthentication) [[2]](https://docs.microsoft.com/en-us/aspnet/web-api/overview/security/basic-authentication))